ui/src/components/avatar/QasAvatar.vue

```diff
@@ -69,21 +42,17 @@ export default {
   },
 
   computed: {
-    backgroundColor () {
-      if (this.color) {
-        return this.color
-      }
-      return this.dark ? this.mainColor : `${this.mainColor}-2`
-    },
-
     classes () {
-      return [this.textColorClass]
+      return [
+        this.dark ? `bg-${this.color}` : `bg-${this.contrastColor}`,
+        this.dark ? `text-${this.contrastColor}` : `text-${this.color}`
+      ]
     },
 
-    color () {
-      return this.$attrs.color
+    contrastColor () {
+      return `${this.color}-contrast`
```

Comment:
É um modifier? `${this.color}--contrast`

Reply:
Não. É uma classe de cor, no padrão do Quasar (`--q-color-primary-contrast`).